logging/logging.go

```diff
@@ -729,25 +729,25 @@ func populateTraceInfo(e *Entry, req *http.Request) bool {
 			return false
 		}
 	}
-	header := req.Header.Get("Traceparent")
+	header := req.Header.Get("X-Cloud-Trace-Context")
```

Comment:
Not sure is it is valid concern, but I wonder if such a change can break existing clients which get used to previous order...

Reply:
> Not sure is it is valid concern, but I wonder if such a change can break existing clients which get used to previous order...

I think this is a valid concern. A change like this for sure can break existing consumers of this library which depend on the old behaviour.

Reply:
In that case, I think we should close this and [save it for v2.0.0](https://github.com/googleapis/google-cloud-go/milestone/15)

Reply:
Makes sense. One additional option is to introduce config parameter controlling which header is taken first - by default such config should prefer existing method to keep backward compatibility. With v.2.0.0 release we could switch default to be a new behavior